app/editor/src/store/slices/content/contentSlice.ts

```diff
@@ -72,6 +72,7 @@ export const contentSlice = createSlice({
             items: action.payload.items.map((content) => castContentToSearchResult(content)),
           }
         : action.payload;
+      saveToLocalStorage('content', state.content?.items ?? []);
```

Comment:
We now store the search results in local storage so that each tab has access to them.